examples/preact/vitest.config.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference types="vitest" />
+
+import { defineConfig } from 'vite'
+
+export default defineConfig({
+  test: {
+    deps: {
+      optimizer: {
+        web: {
+          enabled: true,
+          include: [
```

Comment:
The idea with preact example is to show how to use react dependencies that alias to preact: https://github.com/vitest-dev/vitest/issues/1652

You don't need to include preact deps in optimizer. You need to include dependencies that rely on react so aliasing starts working:

```ts
{
  include: ['react-router-dom']
}
```